stdlib/public/core/VarArgs.swift

```diff
@@ -85,6 +85,30 @@ internal let _registerSaveWords = _countGPRegisters + _countSSERegisters * _sseR
 internal let _countGPRegisters = 16
 @_versioned
 internal let _registerSaveWords = _countGPRegisters
+
+#elseif arch(arm64)
+
+// ARM IHI 0055B
+// va_list may refer to any paramenter may be in one of three memory locations:
+// 1. GP register save area x0 - x7
+// 2. FP/SIMD register save area q0 - q7
+// 3. Stack argument area
+//
+
+@_versioned
+internal let _countGPRegisters = 8
+
+@_versioned
+internal let _countFPRegisters = 8
+
+//128bit, 2 64bit word
+@_versioned
+internal let _fpRegisterWords = 2
```

Comment:
Might be nicer to express this as a `sizeof(double) / sizeof(int)` or something like that.